packages/realm/src/Set.ts

```diff
@@ -129,3 +142,42 @@ export class RealmSet<T = unknown> extends OrderedCollection<T, [T, T]> {
     }
   }
 }
+
+/**
+ * Accessor for getting and setting items in the binding collection, as
+ * well as converting the values to and from their binding representations.
+ * @internal
+ */
+export type SetAccessor<T = unknown> = TypeHelpers<T> & {
+  get: (set: binding.Set | binding.Results, index: number) => T;
+  set: (set: binding.Set, index: number, value: T) => void;
+  insert: (set: binding.Set, value: T) => void;
+};
+
+type SetAccessorFactoryOptions<T> = {
+  realm: Realm;
+  typeHelpers: TypeHelpers<T>;
+  isObjectItem?: boolean;
+};
+
+/** @internal */
+export function createSetAccessor<T>({
```

Comment:
Similarly here - why don't we have a set accessor for `Set<Mixed>`?

Reply:
Will refer to the related [previous thread](https://github.com/realm/realm-js/pull/6513#discussion_r1508894777) 👍